dev/kind/def.bzl

```diff
@@ -41,6 +44,9 @@ attrs = {
     "cluster_name": attr.string(
         mandatory = True,
     ),
+    "kubernetes_version": attr.string(
```

Comment:
The minikube branch uses `k8s_version`; please be consistent (or possibly change the minikube one to match).  I think in the minikube case it's also possible to override this with an environment variable that's set when `bazel` is executed… do the same here?